src/Jackett.Common/Indexers/Torrentseeds.cs

```diff
@@ -28,7 +29,7 @@ private new ConfigurationDataBasicLoginWithRSSAndDisplay configData
             set => base.configData = value;
         }
 
-        public TorrentSeeds(IIndexerConfigurationService configService, WebClient wc, Logger l, IProtectionService ps) :
+        public TorrentSeeds(IIndexerConfigurationService configService, Utils.Clients.WebClient wc, Logger l, IProtectionService ps) :
```

Comment:
Just used it as other indexers are using it like this as well: [TransmitheNet](https://github.com/Jackett/Jackett/blob/889a8da4e5f046d029d86aa34b35f2854c9f67ed/src/Jackett.Common/Indexers/TransmitheNet.cs#L29)
TorrentLeech.cs does this as well

Reply:
Looks like our code base is currently inconsistent then, because [3 indexers](https://github.com/Jackett/Jackett/search?q=%22using+WebClient+%3D+Jackett.Common.Utils.Clients.WebClient%3B%22) use the one I mentioned, [11 indexers](https://github.com/Jackett/Jackett/search?q=%22Utils.Clients.WebClient+wc%22&unscoped_q=%22Utils.Clients.WebClient+wc%22) do it your way, and another [16 indexers](https://github.com/Jackett/Jackett/search?q=%22Utils.Clients.WebClient%22&unscoped_q=%22Utils.Clients.WebClient%22) use some other variation on this method, including qualifying all the way down to `Jackett.Common.Utils.Clients.WebClient` on at least one I saw.

All the more reason for @Jackett/maintainers to decide on a consistent style for future code and legacy cleanup.

Reply:
@cadatoiva I don't care, just pick one and do the changes in all affected indexers so the code is consistent.
Please, don't be so hard on new contributors. He is trying to fix a bug and has taken as reference other parts of our code that are wrong. Try to report only the bugs and if you want to improve the code, you can open another PR with the changes you consider appropriate.
If I have to choose, I prefer more contributors with a tolerable code than the other way around.

Reply:
@ngosang Thanks for your kind words, but I didn't take his words as something preventing me from contributing. 
Actually, I can totally relate to the desire of having consistent code, but as you said or @cadatoiva pointed out, currently there is already some inconsistency in the code base, which makes it difficult to adopt to the "right" code style. But apart from this issue in this specific change request, all his comments were valid and should have been checked by me beforehand.
I have to admit I should have read the contribution guidelines more carefully, but thanks to both of you, you pointed me in the right direction.

Reply:
@ngosang, I want to remind you that at the end of the style guide refactorings (when all the existing code is style guide compliant), we are supposed to be turning on the `dotnet format` check and installing the FxCop NuGet package for the project. This means that at that time, all my comments on style choices become build breaking and PR merge blocking in an automated capacity via those tools. I'm currently just the manual version of the style guide enforcement that can't be turned on with how much needs to change first.

I know we've had a rough couple of weeks, you even more so since this is not the only project you're working on. You're tired and burnt out and don't want to think about syntactic sugar, and I'm probably your least favorite person because I'm nagging about things that you think don't matter. But it's been rough because this project has been ok with "good enough" for years, which lead to us having 2 DOM parsers, and the memory bloat, and the tracker or two that were making 300+ requests on an empty search. We've been needing to reformat the code just to understand what it's trying to do so we can fix these bugs we've fixed recently. We've removed ~100 MB RAM usage just by removing CsQuery.

I know last night with my first review round that I was a blunt, and the tone could have been misconstrued as hostile or aggressive. I was tired, I wanted to get my review in before bed, so I just listed the changes like when you or I review each other's code. I noticed my mistake this morning and [already apologized](https://github.com/Jackett/Jackett/pull/7611#discussion_r391096928) before you decided to correct me. Reminding people that we brought in a style guide on purpose and that we want it followed is not going to alienate developers. If anything, it tends to bring more (and more experienced) developers to the table, because they don't need to do the same amount of work to understand what they're looking at before they start helping out.

Reply:
We both want the same thing, to improve Jackett. I'm not mad at you. I just think we have a lot of open issues and the code can be improved little by little.